gallery/src/components/NxTabs/NxTabsPage.tsx

```diff
@@ -82,37 +80,115 @@ export default function NxTabsPage() {
           <header className="nx-tile-subsection__header">
             <h3 className="nx-h3">NxTabList</h3>
           </header>
-
           <p className="nx-p">
             The parent container for the <code className="nx-code">&lt;NxTab&gt;</code> components.
-            Passes through all attributes to an underlying ul element.
+            Passes through all attributes to an underlying <code className="nx-code">ul</code> element.
           </p>
+          <NxTable className="nx-table--gallery-props">
```

Comment:
I don't think this class does anything

Reply:
> I don't think this class does anything

I removed all instances from `NxTabsPage` but it appears all over the Gallery. https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/b44716991bafacdbfadd26200cce784c7bc998dc